index.html

```diff
@@ -35,10 +35,215 @@ <h1 class="developers-name">Charlie Giorgania</h1>
         </div>
         <div class="colab-detail">
           <div class="colab-butn">
-            <a href="#" class="start-colab">Start collaboration</a>
+            <a href="#" class="start-colab colab-butn colab-detail"
+              >Start collaboration</a
+            >
           </div>
         </div>
       </section>
     </main>
+    <section class="projects-main">
+      <div class="projects-header">
+        <h2 class="main-topic topic-h2">Projects</h2>
+      </div>
+      <div class="grd-container works-catalog">
+        <div class="project-intro">
+          <img
+            src="/resources/images/computer-hands.png"
+            alt="hands on computer"
+          />
+        </div>
+        <div class="works-info">
+          <h3>Project name goes here</h3>
+          <div class="works-skills">
+            <ul class="skills-list">
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="works-bttn">
+              <a href="#" class="works-btn-styles">See this project</a>
+              <img
+                src="/resources/images/arrow.png"
+                alt="arrow image"
+                class="arrow-image"
+              />
+            </div>
```

Comment:
- [ ] Please kindly ensure that the project images are full width as specified in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164580422-eee34925-2bec-418c-a2a2-a0fc504d85e5.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164580469-88210e35-f8fc-4d77-bd15-52c86c931e20.png)